### ISSUE TITLE
about the NELL dataset

### ISSUE THREAD

**Issue by Ashappyboy** [salesforce/MultiHopKG#19]
hi,
![image](https://user-images.githubusercontent.com/31423097/90736164-3b24e400-e300-11ea-9e30-4734d8f36f8c.png)
just as you said, "train.large.triples = raw.kb -dev.triples" and " train.dev.triples is subset of raw.kb" and "train.dev.large.triples = raw.kb". And I don't sure that "the original data release" from the readme is the raw.kb or not. 
So, I want to ask you what is the meaning of doing that.
And in the readme, I don't understand the meaning of " and the final model to test has to be trained with these two files combined". Could you explain it?

**Comment by todpole3** [comment]
Thanks for bringing this to our attention. 

NELL-995 original data release consists of
- raw.kb
- train.triples
- test.triples
where train.triples is a subset of raw.kb, which does not overlap with test.triples.

For development we sampled a subset of train.triples to form the validation set dev.triples. And we extend the train set by including all triples in raw.kb except those in dev.triples.

For the evaluation on test set we train using all triples in raw.kb.

**Comment by Ashappyboy** [comment]
Hi, thank you for your reply soonly.
At first, I'm confused with the development and evaluation. I feel your idea the mode of the experiment by development and evaluation while not the train, dev, test. And split out that sets. 
I got the meaning, thank you very much.
your work is very great and is helpful to me.
Best wishes!
